### DISCUSSION TITLE
Property bindingPackageScan is not allowed. `yaml-schema: Rest`

### DISCUSSION THREAD

**Discussion by cdia8890** [apache/camel-karavan#1452] [Q&A] [answered]
### Ask a question

Highly appreciate if you can clarify why the property `bindingPackageScan` is not allowed in the YAML Rest Schema.

![image](https://github.com/user-attachments/assets/64f81406-546c-4138-8483-de87d9850bad)

TIA

### Variant

Vs Code Extension

### Container Management (if applicable)

None

### Operating System (if applicable)

Windows

### Version

4.8.0

**Comment by mgubaidullin** [comment, marked as answer]
Karavan VS Code extension does not provide any YAML Rest Schema validators.
Perhaps you have some other extensions installed

**Comment by davsclaus** [comment]
Yeah its the standard yaml schema plugin in VSCode that you may use an old version of the schema. Make sure it uses the same Camel version you use.

It loads from the json schema store
https://www.schemastore.org/json/

**Comment by cdia8890** [comment]
Yeah, thanks for the reply.

Putting `camel.rest.bindingPackageScan` in the application.properties worked!

Thanks again.
